### PR TITLE
fix(ingestion/looker): set project-name for imported_projects views

### PR DESCRIPTION
**Pull request by mohdsiddique**

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 1]
```diff
@@ -246,10 +139,21 @@ class ViewField:
...
+    view_name: Optional[str] = None
     is_primary_key: bool = False
     upstream_fields: List[str] = dataclasses_field(default_factory=list)
 
 
+def create_view_project_map(view_fields: List[ViewField]) -> Dict[str, str]:
+    view_project_map: Dict[str, str] = {}
+    for view_field in view_fields:
+        if view_field.view_name is not None and view_field.project_name is not None:
+            view_project_map[view_field.view_name] = view_field.project_name
```
[the remark this answers is not in the document]
Comment (mohdsiddique) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:156]: view within a model has a unique name, so it wouldn't break as it is getting used by LookerExplore.from_api. 
I added the respective doc string in the function.

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 2]
```diff
@@ -734,7 +691,9 @@ def from_api(  # noqa: C901
                 fields=view_fields,
                 upstream_views=list(
                     ProjectInclude(
-                        project=_BASE_PROJECT_NAME,
+                        project=_BASE_PROJECT_NAME
+                        if view_name not in view_project_map
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:695]: can change this to `view_project_map.get(view_name, _BASE_PROJECT_NAME)`

**Review by hsheth2 — APPROVED** [review]
one minor comment but otherwise LGTM

Just to make sure, the new looker_config.py file is purely a refactoring, and there's no code changes in there right?

**Comment by mohdsiddique** [comment]
> one minor comment but otherwise LGTM
> 
> Just to make sure, the new looker_config.py file is purely a refactoring, and there's no code changes in there right?

Yup, It is just refactoring.

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py (hunk 2 above)
Comment (mohdsiddique) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py]: done

**Pull request merged**